app/src/main/res/values/strings.xml

```diff
@@ -541,7 +541,8 @@ Please report any problems you encounter via Github.</string>
 
     <string name="notification_crash_text">Click to view logs</string>
 
-    <string name="notifications_persistent_channel">Persistent notification</string>
+    <string name="notifications_persistent_channel">Syncthing active</string>
```

Comment:
I think this should use a new `name`, otherwise the existing translations for "Persistent notification" will keep being active.

Reply:
No, I just wanted to rename the channel. I tested it out, it works. No new channel is created, but the display name of the channel has been updated from "Persistent notification" to "Syncthing active", which now fits better.

Reply:
Oh, I understand what you mean, I misread. You are right about the translations, of course.

Reply:
Sure in English there's not going to be a problem, I am thinking about the translation system: Is it smart enough to detect the change and remove all existing translations for the `notifications_persistent_channel` string? Otherwise the existing translations of "Persistent notification" will keep being used, while new translations for "Syncthing active" should be used.

Reply:
Unfortunately I've never cared about the translations yet. I'm not sure how they are done. Are they imported manually? I'll have a look and if it proves to be a problem, I'll either remove the current translations there and/or rename the key, as appropriate.

Reply:
I believe transifex automatically imports/reads this file and the translations then need to be imported/committed manually (e.g. https://github.com/syncthing/syncthing-android/commit/8f324a866dcf93e76abee3439d4d4f30ef007c89). I requested access to German&French translations to check what happened to the string after this merge - maybe anyone else with access could check it too, because I won't be active anymore for a bit over a week past midday today :)